gitbot.rb

```diff
@@ -132,17 +160,25 @@ def launch_test_and_setup_status(repo, pr_head_sha, pr_head_ref, pr_base_ref)
   # check the conditions 1,2 and it they happens run_test
   if context_present == false || pending_on_context == true
     check_for_all_files(repo, pr.number, @file_type)
+    if @changelog_test
+      check_if_changes_files_changed(repo, pr)
+      next
+    end
     next if @pr_files.any? == false
     exit 1 if @check
     launch_test_and_setup_status(repo, pr.head.sha, pr.head.ref, pr.base.ref)
     break
   end
- # we want redo sometimes test on a specific PR number
- # (if the jenkins job get lost) even if the test was ok
+  # we want redo sometimes test on a specific PR number
+  # (if the jenkins job get lost) even if the test was ok
   next if @pr_number.nil?
```

Comment:
once the magic word feature, this part will become depracted, but for moment is ok